contracts/interfaces/IAMMWrapper.sol

```diff
@@ -3,11 +3,22 @@ pragma solidity >=0.7.0;
 
 import "./IStrategyBase.sol";
 
+/// @title IAMMWrapper Interface
+/// @author imToken Labs
 interface IAMMWrapper is IStrategyBase {
-    // Operator events
-    event SetDefaultFeeFactor(uint16 newDefaultFeeFactor);
-    event SetFeeCollector(address newFeeCollector);
-
+    /// @notice Emitted after swap with AMM
+    /// @param source The tag of the contract where the order is filled
+    /// @param transactionHash The hash of the transaction structure
+    /// @param userAddr The address of taker
+    /// @param relayed The hash of the order structure
```

Comment:
Indicate if the transaction is relayed by a relayer?

Reply:
Fixed in af7f2a6